azure-pipelines.yml

```diff
@@ -21,18 +21,21 @@ variables:
   ${{ if startsWith(variables['Build.SourceBranch'], 'refs/tags/') }}:
     versionName: $(Build.SourceBranchName)
   installerBaseFileName: speedrun-timer-installer_$(versionName)
-  ${{ if not(startsWith(variables['Build.SourceBranch'], 'refs/tags/v')) }}:
-    buildConfiguration: 'Experimental'
-  ${{ if startsWith(variables['Build.SourceBranch'], 'refs/tags/v') }}:
-    buildConfiguration: 'Release'
-
+  # ${{ if not(startsWith(variables['Build.SourceBranch'], 'refs/tags/v')) }}:
+  #   buildConfiguration: 'Experimental'
+  # ${{ if startsWith(variables['Build.SourceBranch'], 'refs/tags/v') }}:
```

Comment:
I had some issues with getting the experimental build to work. I can look into it again later.